extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1848,9 +1848,7 @@ export class AmpA4A extends AMP.BaseElement {
   preinstallCallback_(embedWin, ampdoc) {
     const parentAmpdoc = this.getAmpDoc();
     installUrlReplacementsForEmbed(
-      // TODO(#22733): Cleanup `parentAmpdoc` once ampdoc-fie is launched.
-      ampdoc || parentAmpdoc,
-      embedWin,
+      ampdoc,
       new A4AVariableSource(parentAmpdoc, embedWin)
```

Comment:
Is this related to the rest of this PR?

Reply:
Not the `ampdoc || parentAmpdoc` specifically. But since I was here, and it was the same overall tracking issue, I went ahead and fixed.